engine/render/src/render/render_script_camera.cpp

```diff
@@ -274,6 +322,8 @@ namespace dmRender
         {"set_far_z",               RenderScriptCamera_SetFarZ},
         {"get_orthographic_zoom",   RenderScriptCamera_GetOrthographicZoom},
         {"set_orthographic_zoom",   RenderScriptCamera_SetOrthographicZoom},
+        {"get_auto_aspect_ratio",   RenderScriptCamera_GetAutoAspectRatio},
+        {"set_auto_aspect_ratio",   RenderScriptCamera_SetAutoAspectRatio},
```

Comment:
I though initially that it should be is_auto_aspect_ration because this is a checkbox? as in gui: https://defold.com/ref/gui/#gui.is_enabled:node-recursive

But then a conversation poped up in my head that when I implemented https://defold.com/ref/gui/#gui.get_visible:node 
we agree that we always use get, and this `is_` in gui is the only place